releases/release-1.17/release_team.md

```diff
@@ -1,11 +1,11 @@
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
 | Lead |  Guinevere Saenger ([@guineveresaenger](https://github.com/guineveresaenger) / Slack: `@gsaenger`) | Jeff Sica ([@jeefy](https://github.com/jeefy) / Slack: `@jeefy`), Jorge Alarcon ([@alejandrox1](https://github.com/alejandrox1) / Slack: `@alejandrox1`), Maria Ntalla ([@mariantalla](https://github.com/mariantalla) / Slack: `@maria`) |
-| Enhancements | Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@mrbobbytables`) | `Anna Jung ([@annajung](https://github.com/annajung) / Slack `@anna`), Jeremy Rickard ([jeremyrickard](https://github.com/jeremyrickard) / Slack: `@jerickar`), Kristin Martin ([kcmartin](https://github.com/kcmartin) / Slack: `@kcmartin`), Nabarun Pal ([palnabarun](https://github.com/palnabarun) / Slack: `@palnabarun`) |
+| Enhancements | Bob Killen ([@mrbobbytables](https://github.com/mrbobbytables) / Slack: `@mrbobbytables`) | Anna Jung ([@annajung](https://github.com/annajung) / Slack `@anna`), Jeremy Rickard ([jeremyrickard](https://github.com/jeremyrickard) / Slack: `@jerickar`), Kristin Martin ([kcmartin](https://github.com/kcmartin) / Slack: `@kcmartin`), Nabarun Pal ([palnabarun](https://github.com/palnabarun) / Slack: `@palnabarun`) |
 | CI Signal | Alena Varkockova ([@alenkacz](https://github.com/alenkacz) / Slack: `@alenkacz`) | |
 | Bug Triage | Josiah Bjorgaard ([@josiahbjorgaard](https://github.com/josiahbjorgaard) / Slack: `@jb`) |  |
 | Docs | Damini Satya Kammakomati ([@daminisatya](https://github.com/daminisatya) / Slack: `@Damini Satya`) |  |
-| Release Notes | Chris Carty ([@cartyc](https://github.com/cartyc)) / Slack: `@macintoshprime` |  |
+| Release Notes | Chris Carty ([@cartyc](https://github.com/cartyc)) / Slack: `@macintoshprime` | Taylor Dolezal ([@onlydole](https://github.com/onlydole) / Slack `onlydole`), Ramesh Kumar ([@kylix3511](https://github.com/kylix3511) / Slack: `kylix3511`), Bridget Kromhout ([bridgetkromhout](https://github.com/bridgetkromhout)) / Slack: `bridgetkromhout`  |
```

Comment:
I think we swapped taylor for eddie right (since Taylor wants to be a RMA) 

let me know if that's not accurate and I an lgtm and approve

Reply:
@claurence - that is correct! Taylor is my name is RMA is my game 😄

Reply:
@claurence update the PR to reflect the change. @onlydole have fun with RMA!